workflows/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Please remove all Gradle wrapper-related files (gradle-wrapper.jar, gradle-wrapper.properties, gradlew and gradlew.bat). They are already in the project in the root directory.

Reply:
Done